packages/wasm-dpp/src/document/extended_document.rs

```diff
@@ -363,6 +362,9 @@ impl ExtendedDocumentWasm {
 }
 
 impl ExtendedDocumentWasm {
+    #[allow(dead_code)]
+    #[deprecated(note = "This function is marked as unused.")]
+    #[allow(deprecated)]
```

Comment:
Why do we allow deprecated code in this function?

Reply:
https://github.com/dashpay/platform/pull/1756#discussion_r1517134639